streams/transferable/resources/helpers.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+
+// Create a ReadableStream that will pass the tests in
```

Comment:
Nit: in other helper files I've preferred to use `self.X = ...` to make it clearer what is exported. Possibly even hiding non-exported values inside a closure, like https://github.com/web-platform-tests/wpt/blob/master/streams/resources/rs-utils.js does.

Reply:
I will fix this in a follow-up change.